frontend/src/app.tsx

```diff
@@ -4,13 +4,15 @@ import { AppRoute } from 'common/enums';
 import AdminPage from 'components/admin-page/admin-page';
 import Sign from 'components/sign/sign'
 import NotFound from 'components/not-found/not-found';
+import Profile from 'components/profile/profile';
 import Clinics from 'components/clinics/clinics';
 import DoctorsSearch from 'components/doctors-search/doctors-search';
 
 const App: React.FC = () => (
   <Switch>
     <Route path={AppRoute.ADMIN_PAGE} exact component={AdminPage}/>
     <Route path={[AppRoute.SIGN_IN, AppRoute.SIGN_UP]} component={Sign}/>
+    <Route path={AppRoute.MY_PROFILE} component={Profile} />
```

Comment:
```suggestion
    <Route path={AppRoute.USER_PROFILE} component={UserProfile} />
```
Rename component/file/folder and etc to `user profile`